pkg/hostagent/mount.go

```diff
@@ -40,16 +40,21 @@ func (a *HostAgent) setupMount(ctx context.Context, m limayaml.Mount) (*mount, e
 	if err := os.MkdirAll(expanded, 0755); err != nil {
 		return nil, err
 	}
+	// NOTE: allow_other requires "user_allow_other" in /etc/fuse.conf
+	sshOptions := "allow_other"
```

Comment:
I believe @AkihiroSuda asked that this should be called `sshfsOptions` instead because these are not ssh options.

Reply:
Hmm... okay... I'll make that change